.github/workflows/shortlog.yml

```diff
@@ -0,0 +1,23 @@
+name: check contributor list
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - '**'
+      - '!CONTRIBUTING.md'  # e.g. when new contributor adds themselves
+      - '!cylc/flow/__init__.py'  # e.g. when __version__ is changed for a new release
+
+jobs:
+  test:
+    runs-on: 'ubuntu-latest'
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # need to fetch all commits to check contributors
+
+      - name: Check CONTRIBUTING.md
+        uses: oliver-sanders/release-actions/check-shortlog@ed0d896d4e7b304e3b4f3319eb047164bbc3927e
```

Comment:
Draft until https://github.com/cylc/release-actions/pull/13 is merged and we can switch to cylc/release-actions.

Reply:
Done.